pelops/training/utils.py

```diff
@@ -0,0 +1,209 @@
+from pelops.datasets.chip import ChipDataset
+from pelops.utils import SetType
+import os
+import os.path
+
+
+
+def attributes_to_classes(chip_dataset, value_extractor):
+    """Extract a set of attributes from a set of Chips and uses them to make
+    unique classses.
+
+    Args:
+        chip_dataset: A ChipDataset, or other iterable of Chips
+        value_extractor: A callable that takes a chip and returns a tuple
```

Comment:
This doesn't strike out to me as very clear. I understand that value_extractor is a function that references, for example, the function make_model in utils.py. In the future, we might want to revisit this to make it more intuitive.